tests/test_util.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from util import strtobool
```

Comment:
`utils` is a symptom of lack of a proper abstraction. Please, do not use it. Keep the `strtobool` function on the main file `decouple.py` since it's not used anywhere else.